core/shared/src/test/scala/com/softwaremill/sttp/testing/HttpTest.scala

```diff
@@ -21,9 +23,8 @@ trait HttpTest[R[_]]
 
   protected def endpoint: String
 
-  protected val binaryFileHash = "2fab8aa91221d3ce09599386643995fc90554f729739c7082b499ae7a720de30"
-  protected val textFileHash = "c57c912fdd2f88f1e8d09d4514744cf706809dd7e352f9aad3ef9bcb586a46a4"
-  private val textWithSpecialCharacters = "Żółć!"
+  protected val binaryFileHash = "565370873a38d91f34a3091082e63933"
```

Comment:
maybe it would be good to include the name of the hasing alg in the name of the constant (eg `binaryFileMD5Hash`)

Reply:
Done